src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -90,10 +90,13 @@
 (defn total-token-fiat-value
   "Returns the total token fiat value taking into account all token's chains."
   [currency {:keys [market-values-per-currency] :as token}]
-  (let [price                     (get-in market-values-per-currency
-                                          [currency :price]
-                                          (get-in market-values-per-currency
-                                                  [constants/profile-default-currency :price]))
+  (let [price                     (or (get-in market-values-per-currency
+                                              [currency :price])
+                                      (get-in market-values-per-currency
+                                              [constants/profile-default-currency :price])
+                                      ;; NOTE: adding fallback value (zero) in case prices are
+                                      ;; unavailable and to prevent crash on calculating fiat value
+                                      0)
```

Comment:
If prices are unanavailable then maybe we can filter them out (like below) ? Just displaying zero would be an incorrect price I think

Reply:
I don't think we should filter them as they may be custom tokens that have not market value yet. But I agree we should not show 0 as price, maybe add `nil` value and do not show any price in the UI would be a better choice IMO.

Reply:
Yes, we shouldn't be filtering out because the market value is unavailable. To create a good UX, we can discuss with the Design team displaying a warning icon next to the token name to indicate that the token price is unavailable or similar.